<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GuidMagneticSensorSetting</class>
 <widget class="QWidget" name="GuidMagneticSensorSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,8">
   <property name="topMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>The device belongs to optional equipment and is added according to actual requirements. If you choose to add, configure the appropriate parameters, or click next.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QRadioButton" name="radioButton_set">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_ignore">
       <property name="text">
        <string>Do not Add</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="vLayout_content"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
